packages/ui-components/src/Tooltip/Tooltip.module.scss

```diff
@@ -90,17 +90,29 @@ $tooltip-arrow-half-width: 5px;
 // Default style
 // --------------------------------------------------
 .default {
-  @include crl-tooltip-style($color-core-neutral-6, $color-core-neutral-6, $color-base-white);
+  @include crl-tooltip-style(
+    $color-core-neutral-6,
+    $color-core-neutral-6,
+    $color-base-white
+  );
 }
 //
 // Dark style
 // --------------------------------------------------
 .dark {
-  @include crl-tooltip-style($color-base-black, $color-base-black, $color-base-white);
+  @include crl-tooltip-style(
+    $color-base-black,
+    $color-base-black,
+    $color-base-white
+  );
 }
 //
 // Light style
 // --------------------------------------------------
 .light {
-  @include crl-tooltip-style($color-base-white, $color-core-neutral-2, $color-font-3);
+  @include crl-tooltip-style(
+    $color-base-white,
+    $color-core-neutral-2,
+    $color-font-3
+  );
```

Comment:
nit: since we're no longer touching this file, let's revert the formatting changes (I know this was just prettier doing it's thing).

Reply:
Done